gravitee-apim-console-webui/src/management/settings/api-quality-rules-ng/api-quality-rules-ng.component.html

```diff
@@ -0,0 +1,168 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div class="title">
+  <h1>API Quality</h1>
+</div>
+
+<form [formGroup]="apiQualityRulesForm" autocomplete="off">
+  <mat-card class="api-quality-rules__form__card">
+    <mat-card-content>
+      <gio-form-slide-toggle formGroupName="apiReview" class="api-quality-rules__form__card__toggle">
```

Comment:
```suggestion
      <gio-form-slide-toggle formGroupName="apiReview" class="api-quality-rules__form__card__form-field">
```
to have one per line.

Reply:
Done